bsp4s/src/main/scala/ch/epfl/scala/bsp/Bsp.scala

```diff
@@ -622,12 +622,23 @@ object ScalaPlatform {
     parent: Option[BuildTargetIdentifier],
     children: List[BuildTargetIdentifier],
 )
+@JsonCodec final case class BspConnectionDetails(
+    name: String,
+    argv: List[String],
+    version: String,
+    bspVersion: String,
+    languages: List[String]
+)
 
+@JsonCodec final case class DebugSessionParams(
+    targets: List[BuildTargetIdentifier],
+    dataKind: String,
+    data: Json
+)
 
-@JsonCodec final case class BspConnectionDetails(
-  name: String,
-  argv: List[String],
-  version: String,
-  bspVersion: String,
-  languages: List[String]
-)
\ No newline at end of file
+object DebugSessionParamsDataKind {
+  val scalaMainClass = "scala-main-class"
```

Comment:
Let's capitalise the names of these symbols, like we do in `BuildTargetDataKind`:
```scala

object BuildTargetDataKind {
  val Scala = "scala"
  val Sbt = "sbt"
}
```